src/util/amino-acid-types.js

```diff
@@ -1,25 +1,25 @@
 export default {
-  a: {
+  A: {
     name: "Alanine (A)",
     description: ["Hydrophobic", "Non-Polar", "Neutral"]
```

Comment:
Ultimately I think we can move the info in this file into `codon-utils`, but I left it for the AA descriptions for now.